structure/0054-quilt-kotlin-team.md

```diff
@@ -0,0 +1,52 @@
+## Summary
+
+The Quilt Kotlin Team will be responsible for developing and maintaining the
+Quilt Kotlin Libraries (Henceforth QKL). QKL is set of libraries, built off of
+QSL, intended for use in the Kotlin programming language.
+
+
+## Motivation
+
+Giving developers an official way to use Kotlin on Quilt, as well as providing
+wrappers for QSL and the Minecraft codebase for easier use in Kotlin.
+
+
+## Explanation
+
+The Quilt Kotlin Team will be responsible for reviewing pull requests,
+designating new modules, developing, and maintaining QKL. It falls under RFC
+0018 (Technical Teams), and will follow the same standards definedin the RFC
+
+
+## Drawbacks
+
+TBD
+
+
+## Rationale and Alternatives
+
+### Subdivision of QSL
+
+QKL shouldn't be a subdivision of QSL, as QSL is based on Java, it'd be
+difficult for maintainers who don't know Kotlin to be able to maintain QKL.
+Having QKL as a subdivision of QSL would mean that changes in the respective
+QSL modules would have to be copied to the QKL module before merging. This is a
+bad idea, because it forces everyone to know Kotlin to be able to contribute.

```

Comment:
Technically you could request help for the contribution, so imo it's more a problem of the general pool of kotlin users available and willing to help, basically creating a low bus factor in these situations unnecessarily

Reply:
> Technically you could request help for the contribution, so imo it's more a problem of the general pool of kotlin users available and willing to help, basically creating a low bus factor in these situations unnecessarily

That's the idea I was trying to get across